Fix:
sffile.c:38:10: error: implicitly declaring library function 'isprint' with type 'int (int)' [-Werror,-Wimplicit-function-declaration]
sffile.c:55:10: error: type specifier missing, defaults to 'int'; ISO C99 and later do not support implicit int [-Wimplicit-int]
https://savannah.gnu.org/bugs/index.php?63720
--- libs/libsffile/sffile.c.orig	2022-03-11 06:15:53.000000000 -0600
+++ libs/libsffile/sffile.c	2024-06-18 22:39:49.000000000 -0500
@@ -20,6 +20,7 @@
  *================================================================*/
 //Interface to Denemo License:  FSF GPL version 3 or later
 
+#include <ctype.h>
 #include <stdio.h>
 #include <string.h>
 #include <stdlib.h>
@@ -51,7 +52,7 @@
 int  ParseSoundfont(char *soundfont, int index, char **name, int *preset, int *bank) {
   FILE *fp;
   static SFInfo sf;
-  static initialized = FALSE;
+  static int initialized = FALSE;
   int i;
   int number = 0;
   if(soundfont) {
